src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIService.java

```diff
@@ -203,6 +203,7 @@ public String checkIfProjectExists(String projectKey, String projectName) {
      */
     @Override
     public boolean checkIfBuildPlanExists(String projectKey, String buildPlanId) {
+        // TODO: we should check that the build script in the programming exercises exists, otherwise builds will fail
```

Comment:
Is this a general TODO or related to this? It feels unrelated. Just want to make sure that you didn't forget to handle this.

Reply:
It's unrelated